.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -9,7 +9,7 @@ build:
     - !!buildcommand
       name: 'Assemble ApplicationInsights-Java JARs'
       command: '.scripts/gradle.cmd'
-      arguments: 'assemble copyLibsToGlobalArtifactsFolder'

```

Comment:
why drops this?

Reply:
do we need `copyLibsToGlobalArtifactsFolder`? I think the pipeline is doing it's own coping to artifacts folder?

Reply:
earlier build failed is because can't find task copyToGlobalArtifactsFolder.

Reply:
release pipeline does that so that i can upload jar to maven.  i think it's different from this.. but wait till the build finishes, we will find out.